src/__tests__/app.test.js

```diff
@@ -0,0 +1,9 @@
+import crpytoReducer, { getData } from '../redux/crypto/cryptoSlice';
+
```

Comment:
Test do not cover most components and pure functions, and some cases are failing! Kindly ensure to improve on the following for your test cases:
- [ ] App **has unit tests for several components of the app and some pure functions** in Redux code.
- [ ] App has integration tests using React Testing Library
- [ ] All test cases are passing!

**Failing test:**
![image](https://user-images.githubusercontent.com/12745474/175779570-d2b1eef3-a708-473f-a441-934b71548aeb.png)